index.bs

```diff
@@ -1447,16 +1459,17 @@ The following operations can be invoked by the client in an authenticator sessio
 This operation must be invoked in an authenticator session which has no other operations in progress. It takes the following
 input parameters:
 
-- The caller's RP ID, as determined by the user agent and the client.
+- The caller's [=RP ID=], as optionally decared by the [=[RP]=] and vetted by the user agent and the client, or defaulted to the
+    caller's [=environment settings object/origin=]'s [=effective domain=], c.f., [[#createCredential]].
 - The [=hash of the serialized client data=], provided by the client.
 - The [=[RP]=]'s {{PublicKeyCredentialEntity}}.
-- The user account's {{PublicKeyCredentialEntity}}.
+- The user account's {{PublicKeyCredentialUserEntity}}.
 - The {{PublicKeyCredentialType}} and cryptographic parameters requested by the [=[RP]=], with the cryptographic algorithms
     normalized as per the procedure in [[WebCryptoAPI#algorithm-normalization-normalize-an-algorithm]].
-- A list of {{PublicKeyCredential}} objects provided by the [=[RP]=] with the intention that, if any of these are known to the
-    authenticator, it should not create a new credential.
-- Extension data created by the client based on the extensions requested by the [=[RP]=].
-- The |requireResidentKey| parameter of the |options|.{{MakeCredentialOptions/authenticatorSelection}} dictionary.  
+- An optional list of {{PublicKeyCredential}} objects provided by the [=[RP]=] with the intention that, if any of these are
```

Comment:
Since [CTAP](https://github.com/fido-alliance/fido-2-specs/pull/226/files#diff-ad05d352dbf182e60e46549766441e10R243) refers to these parameters by name, WebAuthn probably should give them the same names.